### PR TITLE
fix: [CI-13178]: Upgraded go version imaged to 1.22.4-alpine3.19

### PR DESCRIPTION
**Pull request by rahkumar56**
Please read [CONTRIBUTING.md](CONTRIBUTING.md) for details on our [code of conduct](CODE_OF_CONDUCT.md), and the process for submitting pull requests to us.

Fixes #

## Proposed Changes

-
-
-

### Checklist

- [ ] Read the [**CONTRIBUTING**](CONTRIBUTING.md) document.
- [ ] Read the [**CODE OF CONDUCT**](CODE_OF_CONDUCT.md) document.
- [ ] Add tests to cover changes.
- [ ] Ensure your code follows the code style of this project.
- [ ] Ensure CI and all other PR checks are green OR
    - [ ] Code compiles correctly.
    - [ ] Created tests which fail without the change (if possible).
    - [ ] All new and existing tests passed.
- [ ] Add your changes to `Unreleased` section of [CHANGELOG](CHANGELOG.md).
- [ ] Improve and update the [README](README.md) (if necessary).
- [ ] Ensure [documentation](./DOCS.md) is up-to-date. The same file will be updated in [plugin index](https://github.com/drone/drone-plugin-index/blob/master/content/meltwater/drone-cache/index.md) when your PR is accepted, so it will be available for end-users at http://plugins.drone.io.

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -63,7 +63,7 @@ steps:
         path: /drone/src/tmp/testdata/cache
 
   # - name: generate
-  #   image: golang:1.22.0-alpine3.11
+  #   image: golang:1.22.4-alpine3.19
```
Comment (devkimittal) [inline .drone.yml:66]: why not just remove this?

Comment (rahkumar56) [reply .drone.yml]: removed commented line of code.

**Comment by devkimittal** [comment]
are we not updating golang version in code as well? go.mod

File: .drone.yml [hunk 2]
```diff
@@ -62,15 +62,6 @@ steps:
       - name: testdata
         path: /drone/src/tmp/testdata/cache
 
-  # - name: generate
```
Comment (raghavharness) [inline .drone.yml:65]: Is this not required?

**Comment by raghavharness** [comment]
Can you update in go.mod as well. Also it's best to run `go mod tidy` after updating the go.mod file

**Review by HenAmar — APPROVED** [review]

**Review by HenAmar — APPROVED** [review]

File: .drone.yml (hunk 2 above)
Comment (rahkumar56) [reply .drone.yml:65]: Yes not required and Devki asked me to remove this

**Pull request merged**